Products/ZenReports/reports/Performance_Reports/Availability_Report.rpt

```diff
@@ -21,8 +21,10 @@
                          zem.defaultAvailabilityEnd());
     objects python:zem.getAvailability(tableState);
     batch python:tm.getBatch(tableName,objects);
-	exportFields python:['device', 'systems', 'availability'];
-	">
+    exportFields python:[('title', 'Device'),
```

Comment:
Does this work OK on devices without a title? Usually we use device.titleOrId() for things like this.